compiler/src/dotty/tools/dotc/util/Signatures.scala

```diff
@@ -42,25 +42,21 @@ object Signatures {
   /**
    * Extract (current parameter index, function index, functions) out of a method call.
    *
-   * @param path The path to the function application
+   * @param enclosingApply Enclosing function application
    * @param span The position of the cursor
    * @return A triple containing the index of the parameter being edited, the index of the function
    *         being called, the list of overloads of this function).
    */
-  def callInfo(path: List[tpd.Tree], span: Span)(using Context): (Int, Int, List[SingleDenotation]) =
-    path match {
-      case UnApply(fun, _, patterns) :: _ =>
-        callInfo(span, patterns, fun, Signatures.countParams(fun))
-      case Apply(fun, params) :: _ =>
-        callInfo(span, params, fun, Signatures.countParams(fun))
-      case _ =>
-        (0, 0, Nil)
-    }
+  def callInfo(enclosingApply: Option[tpd.Tree], span: Span)(using Context): (Int, Int, List[SingleDenotation]) =
+    enclosingApply.map {
```

Comment:
Can we do the `find`  (that we do below in `Interactive.pathTo(trees, pos).find {`) here so that no change of signature is needed? I avoid changing the signature whenever possible.

Reply:
Done